yazi-adapter/src/image.rs

```diff
@@ -31,17 +32,36 @@ impl Image {
 				encoder.write_image(&rgba, rgba.width(), rgba.height(), ExtendedColorType::Rgba8)?;
```

Comment:
Should the same process of sRGB to RGB(A) conversion be applied to the PNG encoder as well for transparent images?

Reply:
I didn't do it there as I don't know of a case where that affects images with alpha, therefore I don't have how to test it.

Reply:
Ok, I think I found some, I'll give it a try.